public/static/locales/en/common.json

```diff
@@ -26,49 +27,50 @@
     "filter": "Filter",
     "help": "Help",
     "home": "Home",
-    "intercomAriaLabel": "Chat with CyVerse support",
     "interactiveAnalysisUrl": "{{message}}. Access your running analysis here.",
-    "searching": "Searching...",
+    "intercomAriaLabel": "Chat with CyVerse support",
     "login": "Login",
     "logout": "Logout",
     "manageAccount": "Manage your CyVerse Account",
     "markAsRead": "Mark All as Read",
-    "moveUp": "Move Up",
-    "moveDown": "Move Down",
     "more": "More",
+    "moveDown": "Move Down",
+    "moveUp": "Move Up",
     "newNotificationAriaLabel": "Show New Notifications",
     "next": "Next",
-    "notifications": "Notifications",
     "noNotifications": "No Notifications to display.",
+    "noOptions": "No results to display.",
+    "noRunningVice": "No currently running VICE jobs",
```

Comment:
Might want to use `analyses` instead of `jobs`. May be `No running VICE analyses. `